### ISSUE TITLE
assign Ensemble gene ids to transcripts

### ISSUE THREAD

**Issue by HadrienG** [HadrienG/tutorials#11]
in the RNA seq tutorial in this code block

```R
txdb <- makeTxDbFromGFF("chr22_genes.gtf")
k <- keys(txdb, keytype = "GENEID")
tx2gene <- select(txdb, keys = k, keytype = "GENEID", columns = "TXNAME")
head(tx2gene)
```

`GENEID` and `TXNAME` should be swapped

**Comment by TheMuffinMan563** [comment]
This is a poor tutorial with many reference file not being provided. Where can we get chr22 fasta files, what's the samples.txt suppose to contain in "samples <- read.table("samples.txt", header = TRUE)", what's the mapping rates for salmon quant.

**Comment by HadrienG** [comment]
Both the chr22 fasta file and the sample.txt file are in the `toy_rna.tar.gz` archive.

Thank you for your constructive feedback 😊

**Comment by TheMuffinMan563** [comment]
Yeah sorry later on I realized the manual download link for FASTQ doesn't have it whereas curl via curl -O -J -L https://osf.io/7zepj/download has all the necessary elements. I didn't know they would be different.
